src/pages/verify/integrations/nodejs-express.mdx

```diff
@@ -33,274 +20,240 @@ Once you register your application you will also need some of the information fo
 
-```javascript
-passport.use(
-  'oidc',
-  new Strategy({ client }, (tokenSet, userinfo, done) => {
-    return done(null, tokenSet.claims());
-  })
-);
-```
-
-The last thing you need to finish setting up the passport is to provide a function to serialize and deserialize a user. Add the following code to your app.js file.
-
-```javascript
+app.use(passport.initialize());
+app.use(passport.session());
 passport.serializeUser(function(user, done) {
   done(null, user);
 });
 passport.deserializeUser(function(user, done) {
   done(null, user);
 });
-```
-
-## Trigger authentication in your application
-
-### Choosing the specific login method
-
-import LoginMethodsSnippet from '../../../snippets/login-methods-and-path-encoded.mdx';
-
-<LoginMethodsSnippet />
 
-In the sample, we have created a **form** which will put the chosen login option in a request parameter and redirect a user to the authentication route specified.
-
-```html
-<select class="form-control" name="loginmethod" form="loginForm">
-  <option value="urn:grn:authn:no:bankid">Norwegian BankID</option>
-  <option value="urn:grn:authn:no:vipps">Norwegian Vipps</option>
-  <option value="urn:grn:authn:se:bankid:same-device">Swedish BankID same device</option>
-  <option value="urn:grn:authn:se:bankid:another-device">Swedish BankID another device</option>
-  <option value="urn:grn:authn:dk:nemid:poces">Danish NemID personal with code card</option>
-  <option value="urn:grn:authn:dk:nemid:moces">Danish NemID employee with code card</option>
-  <option value="urn:grn:authn:dk:nemid:moces:codefile">Danish NemID employee with code file</option>
-  <option value="urn:grn:authn:fi:bank-id">Finish e-ID - BankID</option>
-  <option value="urn:grn:authn:fi:mobile-id">Finish e-ID - Mobile certificate</option>
-  <option value="urn:grn:authn:fi:all">Finish all options</option>
-</select>
-
-<form action="/auth" id="loginForm">
-  <input class="btn btn-outline-primary" type="submit" value="Login">
-</form>
-```
-
-### Start the authentication request
-
-After you submit a form, you will be redirected to the `/auth` route. Now you have to set up an Express middleware for the `/auth` route where you will tell passport to start the authentication request and redirect the user to the chosen login option.
-
-To do that, you have to invoke `authenticate` method and tell it which strategy to use. You also have to provide an object with `acr_values` and optionally some other additional parameters.
-
-Add the following code to your app.js file.
+const redirectPassport = new CriiptoVerifyRedirectPassportStrategy(
+  {
+    domain: "{{YOUR_CRIIPTO_DOMAIN}}",
+    clientID: "{{YOUR_CLIENT_ID}}",
+    clientSecret: "{{YOUR_CLIENT_SECRET}}",
+    // Should match an express route that is an allowed callback URL in your application
+    // This route should also have the authentication middleware applied.
+    redirectUri: '/login',
+    postLogoutRedirectUri: '/',
+
+    // Ammend authorize request if you wish
```

Comment:
Ammend -> amend (2 occurencies)

Reply:
I'm almost certain I make that typo 100% of the time :D